modules/combined/tests/generalized_plane_strain_tm_contact/generalized.exodiff

```diff
@@ -0,0 +1,33 @@
+COORDINATES absolute 1.e-6    # min separation not calculated
+
+TIME STEPS relative 1.e-6 floor 0.0     # min:               0 @ t1 max:               2 @ t11
+
+GLOBAL VARIABLES relative 1.e-6 floor 0.0
+	max_strain_zz  # min:               0 @ t1	max:     0.023318442 @ t11
+	min_strain_zz  # min:               0 @ t1	max:     0.023318437 @ t11
+	react_z        floor 1e-4 # min:               0 @ t1	max:   2.1908779e-06 @ t9
+
+NODAL VARIABLES relative 1.e-5 floor 1e-10
+	contact_pressure  # min:               0 @ t1,n1	max:       9991.7939 @ t11,n4
+	disp_x            # min:               0 @ t1,n1	max:      0.02937928 @ t11,n14
+	disp_y            # min:               0 @ t1,n1	max:     0.044828721 @ t11,n10
+	nodal_area_mech   # min:               0 @ t1,n2	max:      0.50738456 @ t11,n4
+	paired_temp       # min:               0 @ t1,n1	max:       1.1375322 @ t11,n1
+	penetration       floor 1e-9 # min:               0 @ t1,n1	max:     0.016005443 @ t2,n1
+	strain_zz         # min:               0 @ t1,n1	max:     0.023318442 @ t11,n12
+	temp              # min:               0 @ t1,n1	max:               2 @ t11,n10
+
+ELEMENT VARIABLES relative 1.e-5 floor 1e-10
+	stress_xx      # min:               0 @ t1,b1,e1	max:       12948.737 @ t11,b1,e2
+	stress_xy      # min:               0 @ t1,b1,e1	max:       2009.6183 @ t11,b2,e5
+	stress_yy      # min:               0 @ t1,b1,e1	max:       2975.5513 @ t11,b2,e8
+	stress_zz      # min:               0 @ t1,b1,e1	max:       17721.946 @ t11,b1,e4
+	strain_xx      # min:               0 @ t1,b1,e1	max:     0.033473637 @ t11,b2,e5
+	strain_xy      # min:               0 @ t1,b1,e1	max:    0.0026125038 @ t11,b2,e5
+	strain_yy      # min:               0 @ t1,b1,e1	max:     0.041654745 @ t11,b2,e7
+	aux_strain_zz  # min:               0 @ t1,b1,e1	max:     0.023318441 @ t11,b2,e7
+
+# No NODESET VARIABLES
+
+# No SIDESET VARIABLES
+
```

Comment:
Why does every solid mechanics test now have to have one of these files?  Does tis test not work without it?

These files are piling up and adding a lot of maintenance overhead.  It would be great to restrict these files to only where they are necessary.